server/package.json

```diff
@@ -96,6 +101,12 @@
     "undoAllMigrate": "sequelize db:migrate:undo:all",
     "undoLastMigrate": "sequelize db:migrate:undo"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged",
+      "pre-push": "ng build --aot true"
+    }
+  },
```

Comment:
@Dmac26 I think we had decided we didn't want to run this on commits so let's clean out the husky related code.

Reply:
@briandavidson - It looks like missed the husky stuff on the server side. Removed